MUXSDKKaltura/MUXSDKKaltura/MUXSDKStats.swift

```diff
@@ -173,11 +173,37 @@ public class MUXSDKStats: NSObject {
      */
     public static func dispatchErrorForPlayer(name: String, code: String, message: String) {
         guard let binding = bindingsManager.bindings[name] else {
+            print("MUXSDK-WARNING - Player binding not found for player name: \(name).")
             return
         }
         
         binding.dispatchError(code: code, message: message)
     }
+    
+    /**
+     Allows customerData to be set or updated for the player
+     
+     Use this method after you have already initialized the Mux SDK at any time before the player has been destroyed.
+     
+     - Parameters:
+     - name: The name of the player to update
+     - customerData: A MUXSDKCustomerData object with player, video, and view metadata
+     */
+    public static func setCustomerDataForPlayer(name: String, customerData: MUXSDKCustomerData) {
```

Comment:
The behavior I see for this is that it will only update the data that was nil so for example if I send a new title for video data and it already had one it wont update. Shouldn't we specify this if it's the expected behavior? I confirmed we have the same behavior in avPlayer. Cause I kind of was expecting it to update anything from this description, like overwrite old values if I sent something new different than nil

Reply:
Just noting what we talked about in slack: we confirmed that this behavior is expected  - the rollup strategy used by the processors for video title is "first" and therefore only the first value is saved